lua/modules/uikit.lua

```diff
@@ -2864,6 +2957,7 @@ function createUI(system)
 		end
 
 		local pressed = pressed
+		pressed._dragging = true
```

Comment:
Should move in the block below, `pressed` could be `nil`

Reply:
Also, there's a `CLICK_MOVE_SQR_EPSILON` in `controls`. Same logic should be used here. 
On mobile, very small, unwanted drag events can happen a lot (depending on devices)

Reply:
I think the best solution is to change how the PointerDrag is triggered.
Today, controls listen to PointerDrag and calls Pointer.DragBegin/Pointer.Drag/Pointer.DragEnd
which means that listening on PointerDrag or adding a Pointer.Drag callback is not the same behavior.

I think the best solution is to have a SystemPointerDrag listened by controls, and this calls Pointer.Drag + send PointerDrag events for games to avoid this issue, as I think this should not be handled individually.

Let me know if I should create an issue, not sure it goes in this PR.